gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/HealthChecker.java

```diff
@@ -34,7 +34,7 @@ public HealthChecker(Notifier notifier)
     public void observe(List<ClusterStats> clustersStats)
     {
         for (ClusterStats clusterStats : clustersStats) {
-            if (!clusterStats.healthy()) {
+            if (clusterStats.healthy() == TrinoHealthStateType.UNHEALTHY) {
```

Comment:
nitpick: feels weird to read `does healthy() = UNHEALTHY ?`, would `healthState() == UNHEALTHY` read better?

Reply:
or maybe just `clusterStats.Health()` ?

Reply:
Sure, open to whichever naming - seeing the ‘y’ suffix (to me) on healthy implies boolean

Reply:
I like `healthState() == UNHEALTHY`